version_control/katalyst_by_codurance/css/elements/_buttons.css

```diff
@@ -57,6 +57,37 @@ input[type="submit"]:disabled, {
   color: #E6E6E6;
 }
 
+/* Floating button */
```

Comment:
Yes, I thought about that but because we have helper classes for buttons I thought it was the best place to have it instead of creating the helper class only accesed within the module.